### ISSUE TITLE
Selection via keyboard

### ISSUE THREAD

**Issue by N-R-K** [resurrecting-open-source-projects/scrot#369]
Since scrot is aimed at cli users, it's a fair assumption that a decent amount of scrot's users may prefer using the keyboard over mouse.

And so being able to select via keyboard could be a nice improvement.

**Comment by hew02** [comment]
Just to confirm, when you say selection you are referring to window selection correct?

**Comment by N-R-K** [comment]
> when you say selection you are referring to window selection correct?

No, area selection is possible with keyboard too. You can take a look at [selx](https://codeberg.org/NRK/selx) to see how it can be done if you're interested in working on this.

**Comment by hew02** [comment]
Oh I see, nice! Yeah selx feels great. I'd be down to work on this, started using scrot and it's perfectly simple. So should this encompass both area selection and window select?
